jjb/3rdparty/3rdparty.yaml

```diff
@@ -0,0 +1,132 @@
+---
+- project:
+    name: 3rdparty-boost-vfx2018
+    3rdparty: boost
+    download: !include-raw-escape: boost/download.sh
+    jobs:
+      - '3rdparty-{3rdparty}-stage-{stream}'
+    project: ci-management
+    project-name: ci-management
+    build-node: ubuntu1604-builder-docker-2c-2g
+    stream: vfx2018
+
+- project:
+    name: 3rdparty-tbb-vfx2018
+    3rdparty: tbb
+    download: !include-raw-escape: tbb/download.sh
+    jobs:
+      - '3rdparty-{3rdparty}-stage-{stream}'
+    project: ci-management
+    project-name: ci-management
+    build-node: ubuntu1604-builder-docker-2c-2g
+    stream: vfx2018
+
+- project:
+    name: 3rdparty-ilmbase-vfx2018
```

Comment:
We have staging builds already for openexr artifacts. Can we not use what already exists?